internal/pkg/describe/webapp.go

```diff
@@ -205,6 +198,25 @@ func (d *WebAppDescriber) ECSParams(envName string) (*WebAppECSParams, error) {
 	}, nil
 }
 
+// GetServiceArn returns the ECS service ARN of the application in an environment.
+func (d *WebAppDescriber) GetServiceArn(envName string) (*ServiceArn, error) {
+	env, err := d.store.GetEnvironment(d.app.Project, envName)
+	if err != nil {
+		return nil, err
+	}
+	appResources, err := d.describeStackResources(env.ManagerRoleARN, env.Region, stack.NameForApp(d.app.Project, env.Name, d.app.Name))
```

Comment:
How hard would it be to search by tag, rather than stack?

Reply:
Have no idea if we can do it easily since "DescribeStackResources" can only filter by stack name. Also, why do we want to search it by tag?

Reply:
I'm just keeping terraform support in the back of my mind.

Reply:
OK then it starts haunting me now